maskrcnn_benchmark/modeling/roi_heads/mask_head/inference.py

```diff
@@ -119,25 +120,28 @@ def paste_mask_in_image(mask, box, im_h, im_w, thresh=0.5, padding=1):
     padded_mask, scale = expand_masks(mask[None], padding=padding)
     mask = padded_mask[0, 0]
     box = expand_boxes(box[None], scale)[0]
-    box = box.numpy().astype(np.int32)
+    box = box.type(dtype=torch.int32)
 
     TO_REMOVE = 1
     w = box[2] - box[0] + TO_REMOVE
     h = box[3] - box[1] + TO_REMOVE
     w = max(w, 1)
     h = max(h, 1)
 
-    mask = Image.fromarray(mask.cpu().numpy())
-    mask = mask.resize((w, h), resample=Image.BILINEAR)
-    mask = np.array(mask, copy=False)
+    # Set shape to [batchxCxWxH]
+    mask = mask.expand((1, 1, -1, -1))
+
+    # Resize mask
+    mask = mask.type(torch.float32)
```

Comment:
Can we use instead `mask.to(torch.float32)` here? It's the new API for it, and `.type` is kind of deprecated.